sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2657,15 +2650,15 @@
           "type": "string"
         },
         "outputBlobContainerUri": {
-          "description": "URI containing SAS token to a blob container.  This is used to output the status of the job and the results.",
+          "description": "URI containing SAS token to a blob container. This is used to output the status of the job and the results.",
```

Comment:
SAS token to access a blob container? SAS itself means URI - A shared access signature (SAS) is a URI that grants restricted access rights to Azure Storage resources.

Reply:
Good call. Will update.

Reply:
For now, I'll write "...**a** blob container" as opposed to something else like "... **your** blob container" until we finalize on the discussion tomorrow.